azure-pipelines.yml

```diff
@@ -27,36 +28,38 @@ stages:
             displayName: 'Setup TMP Directory'
           - checkout: self
             clean: true
-            submodules: false
+            submodules: true

```

Comment:
Why do we need to check out submodules? We shouldn't need them for the clang-format/validate build, and we have very special handling of llvm-project and boost-math for the actual build.

Reply:
we need boost-math because otherwise we can't configure the project.

Reply:
fixed by making format a full on project